graphix/clifford.py

```diff
@@ -287,6 +293,17 @@ def measure(self, pauli: graphix.pauli.Pauli) -> graphix.pauli.Pauli:
         symbol, sign = table[pauli.symbol.value]
         return pauli.unit * graphix.pauli.TABLE[symbol + 1][sign][False]
 
+    @classmethod
+    def __get_pydantic_core_schema__(
+        cls, source_type: typing.Any, handler: pydantic.GetCoreSchemaHandler
+    ) -> pydantic_core.CoreSchema:
+        def check_clifford(obj) -> Clifford:
+            if not isinstance(obj, Clifford):
+                raise ValueError("Clifford expected")
+            return obj
+
+        return pydantic_core.core_schema.no_info_plain_validator_function(function=check_clifford)
```

Comment:
are we sure we want to do this? does not seem to be the most stable feature of pydantic, see https://docs.pydantic.dev/latest/concepts/types/#customizing-validation-with-__get_pydantic_core_schema__